lighthouse-core/test/results/sample_v2.json

```diff
@@ -1854,6 +1854,34 @@
         ]
       }
     },
+    "no-unload-listeners": {
+      "id": "no-unload-listeners",
+      "title": "Listens for the `unload` event",
+      "description": "The `unload` event does not fire reliably and listening for it can prevent browser optimizations like the back/forward cache. Consider using the `pagehide` or `visibilitychange` events instead. [Learn More](https://developers.google.com/web/updates/2018/07/page-lifecycle-api#the-unload-event)",
+      "score": 0,
+      "scoreDisplayMode": "binary",
+      "details": {
+        "type": "table",
+        "headings": [
+          {
+            "key": "source",
+            "itemType": "source-location",
+            "text": "URL"
+          }
+        ],
+        "items": [
+          {
+            "source": {
+              "type": "source-location",
+              "url": "(unknown)",
```

Comment:
is this expected, is there a real scriptId we can use?

Reply:
> is this expected, is there a real scriptId we can use?

it's because the `JsUsage` sample artifacts are an empty array, so there's no real scriptId to match up regardless. I think this might be why some unit tests are failing as well. Going to take a look.

Reply:
> it's because the `JsUsage` sample artifacts are an empty array, so there's no real scriptId to match up regardless. I think this might be why some unit tests are failing as well. Going to take a look.

ok, this is two parts:
- updated the sample `artifacts.json` to have the bare minimum `JsUsage` info - scripts but no function coverage entries (someone who wants to have those in the sample artifacts for real can add it :)
- `source-location` really doesn't like when `url` isn't actually a `url`, which is what was causing the report unit test failures. When the url is unknown, switched over to `renderTextUrl`